src/web/server/document.tsx

```diff
@@ -105,6 +105,11 @@ export const document = ({ data }: Props) => {
 
     const { openGraphData } = CAPI;
     const { twitterData } = CAPI;
+    const keywords =
+        typeof CAPI.config.keywords === 'undefined' ||
+        CAPI.config.keywords === 'Network Front'
```

Comment:
I believe DCR doesn't currently do fronts but since it's ramping up I figured it won't be long until it does 🤞 and decided to leave this in in antecipation.